src/H5FDs3comms.c

```diff
@@ -941,7 +941,7 @@ H5FD_s3comms_s3r_getsize(s3r_t *handle)
      * PARSE RESPONSE *
      ******************/
 
-    start = HDstrstr(headerresponse, "\r\nContent-Length: ");
+    start = HDstrcasestr(headerresponse, "\r\nContent-Length: ");
```

Comment:
Note that the read-only S3 VFD is currently neither built nor tested via GitHub actions at this time. We should ask @lrknox to test this independently (even though it's a trivial change).

This is also a reminder to us to set up ros3 VFD testing in the very near future.

Reply:
In addition to the regular requirements, I needed to install `libcurl4-openssl-dev` to build and test - in case that helps :-)

Reply:
Tested berghaus:develop with ros3 enabled and S3 bucket for tests configured.  All tests passed.